shell.nix

```
@@ -23,6 +23,7 @@ let
     (pkgs.writeScriptBin "install-editor" ''
       #!/usr/bin/env bash
       set -e
+      ${node}/bin/npm --scripts-prepend-node-path=true install
```

```suggestion
      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)
      ${node}/bin/npm --scripts-prepend-node-path=true install
```
I'm pretty sure we'd need to `cd` here to ensure we're in the right directory first

---

thanks! fixed :)